src/components/Recipes/RecipeItem.tsx

```diff
@@ -38,11 +39,32 @@ const RecipeItem: React.FC<{recipeID: number, text: string, image: string, }> =
     });
   };
 
+  const searchResult = searchResultInfo.filter((recipeItem) => props.recipeID === recipeItem.id)[0]
+
+  const missingIngredientList = searchResult.missedIngredients.map((missedIngredient) => {
+    return <li key={missedIngredient.id}>{missedIngredient.amount} {missedIngredient.unitLong} {missedIngredient.name}</li>
+  });
+
+  const usedIngredientList = searchResult.usedIngredients.map((usedIngredient) => {
+    return <li key={usedIngredient.id}>{usedIngredient.amount} {usedIngredient.unitLong} {usedIngredient.name}</li>
+  });
+
   return (
     <li key={props.recipeID}>
       <Card class='recipeCard'>
         <img className="card-header" src={props.image} />
         <h2>{props.text}</h2>
+
+        <div className="missingIngredients">
+          <p>{searchResult.missedIngredientCount} {searchResult.missedIngredientCount > 1 && searchResult.missedIngredientCount !== 0 ? `Missing Ingredients:` : `Missing Ingredient:`}</p>
+          <ul>{missingIngredientList}</ul>
+        </div>
+        
+        <div className="usedIngredients">
```

Comment:
why arent we reusing the same class here, looks like theyre the same?

not an absolute rule, but DRY (don't repeat yourself) code should be the default, and only repeat when necessary not convenient

Reply:
It was initially done in case they would ultimately be styled differently to set them apart (like font color) down the road. It is currently unnecessary, however